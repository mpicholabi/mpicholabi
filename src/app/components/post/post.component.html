<h1>{{ title }}</h1>
<p class="text-red-500">post works!</p>
