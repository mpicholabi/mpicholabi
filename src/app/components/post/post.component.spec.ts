import { TestBed, ComponentFixture } from '@angular/core/testing';
import { PostComponent } from './post.component';

describe('Test PostComponent', () => {
  let component: PostComponent;
  let fixture: ComponentFixture<PostComponent>;

  beforeAll(async() => {
    TestBed.configureTestingModule({
      declarations: [PostComponent],
    });
    fixture = TestBed.createComponent(PostComponent);
  });

  it('Should create component', () => {
    expect(fixture.componentInstance).toBeDefined();
  });

  it('Should have title', () => {
    expect(fixture.componentInstance.title).toContain('Test');
  });

  it('Should generate component with snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
