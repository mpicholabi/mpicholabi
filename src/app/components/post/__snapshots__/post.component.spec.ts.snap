// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Should generate component with snapshot 1`] = `
<app-post
  title={[Function String]}
>
  <h1>
  </h1><p
    class="text-red-500"
  >
    post works!
  </p>
</app-post>
`;

exports[`Test PostComponent Should generate component with snapshot 1`] = `
<app-post
  title={[Function String]}
>
  <h1>
  </h1><p
    class="text-red-500"
  >
    post works!
  </p>
</app-post>
`;
